components/datatable/Datatable.jsx

```diff
@@ -270,6 +315,27 @@ const Datatable = ({
           )}
         </div>
       </div>
+      {downloadCSV && (
+        <div className='d-flex justify-center'>
+          <button
+            className='btn btn-primary d-flex items-center justify-between gap-1'
+            onClick={() => {
+              try {
+                CSVDownloader(jsonToCSV(data), CSVName);
```

Comment:
If you are using the same CSVDownloader component that we used earlier then you don't need to convert jsonToCSV, the component will do it automatically.